dsc/examples/PSConfEU_2024_Demo.md

```diff
@@ -0,0 +1,103 @@
+# PSConfEU 2024 Demos
+
+## DSC Command-line
+
+### List DSC resources
+
+- List DSC resources
+
+```powershell
+dsc resource list
+```
+
+- List PowerShell Adapted DSC resources
+
+```powershell
+dsc resource list --adapter Microsoft.DSC/PowerShell
+```
+
+- Find DSC resources
+
+```powershell
+dsc resource list '*DSC*'
+```
+
+### DSC Resource Invocation
+
+- Invoke OSInfo DSC resource
+
+```powershell
+dsc resource get -r Microsoft/osinfo
+```
+
+### Output formats
+
+- YAML
+- JSON
+- Pretty-JSON
+
+### Tracing
+
+- Error, Warn, Info, Debug, Trace
+- Logging to ETW/Syslog and files
+
+### WhatIf
+
+## DSC Resources
+
+### Resource Manifest
+
+- JSON vs YAML
+- Args, JSON Input, Env Var
+- JSON Schema
+
+### Resource Types
+
+- Resource
+- Group
+- Adapter
+- Import
```

Comment:
We probably want to be careful about calling these kinds, since `type` in the syntax and usage means "specific resource."

```suggestion
### Resource Kinds

- Resource
- Group
- Adapter
- Import
```